core/src/trezor/lvglui/scrs/homescreen.py

```
@@ -4747,28 +4809,42 @@ def on_click(self, event_obj):
+                            workflow.spawn(
+                                fingerprints.request_delete_fingerprint(
+                                    prompt,
+                                    on_remove=lambda: self.on_remove_group(group),
+                                )
+                            )
+                        else:
+                            fp_id = self.fingerprint_list[i]
+                            prompt = _(i18n_keys.FORM__FINGER_STR).format(i + 1)
+                            workflow.spawn(
+                                fingerprints.request_delete_fingerprint(
+                                    prompt, on_remove=lambda: self.on_remove(fp_id)
```

Ruff (0.8.2)

Function definition does not bind loop variable `fp_id`

(B023)